Formula/afl-fuzz.rb

```diff
@@ -22,6 +22,8 @@ class AflFuzz < Formula
   def install
     system "make", "PREFIX=#{prefix}", "AFL_NO_X86=1"
     system "make", "install", "PREFIX=#{prefix}", "AFL_NO_X86=1"
+
+    on_linux { rm share/"afl/testcases/others/elf/small_exec.elf" }
```

Comment:
Is this being deleted because of compiler shims?  If so, we should add a comment explaining that.

Reply:
Was an audit failure https://github.com/Homebrew/homebrew-core/runs/3156084206 for incompatible architecture in ELF file.
```
==> FAILED
Error: 1 problem in 1 formula detected
Error: No such file or directory @ realpath_rec - /home/runner
afl-fuzz:
  * Binaries built for an incompatible architecture were installed into afl-fuzz's prefix.
    The offending files are:
      /home/linuxbrew/.linuxbrew/Cellar/afl-fuzz/2.57b_1/share/afl/testcases/others/elf/small_exec.elf
```

Seemed to be only a testcase file, so probably safe to delete.

I'll add a comment relating to reason.

Reply:
I'm assuming the incompatible architecture is 32-bit x86?

Reply:
I added details in comment listing `elf32-i386` arch.

Reply:
Let's remove this everywhere, as it looks like the kind of file that gets installed regardless of the platform.

Reply:
Looks like the directory structure is different on macOS:
```console
❯ fd small_exec.elf /usr/local/opt/afl-fuzz
/usr/local/opt/afl-fuzz/share/afl/others/elf/small_exec.elf
```

Seems to be BSD vs GNU implementation of `cp -r`:
```
ln -sf afl-as ${DESTDIR}/usr/local/Cellar/afl-fuzz/2.57b_1/lib/afl/as
install -m 644 docs/README docs/ChangeLog docs/*.txt ${DESTDIR}/usr/local/Cellar/afl-fuzz/2.57b_1/share/doc/afl
cp -r testcases/ ${DESTDIR}/usr/local/Cellar/afl-fuzz/2.57b_1/share/afl
cp -r dictionaries/ ${DESTDIR}/usr/local/Cellar/afl-fuzz/2.57b_1/share/afl
```